webapp/src/components/cardDetail/__snapshots__/cardDetailContents.test.tsx.snap

```diff
@@ -597,19 +597,19 @@ exports[`components/cardDetail/cardDetailContents should match snapshot after on
         class="octo-block-margin"
       />
       <div
-        class="MarkdownEditor octo-editor  active"
+        class="MarkdownEditor octo-editor  "
```

Comment:
I don't think this file needs change. I think you can uncommit the changes in this file.